tinker/templates/admin/redirects/home.html

```diff
@@ -169,13 +172,15 @@
                     $.post(url, data, function (data) {
                         loadTableForSearch($('#new-redirect-from').val(), 'from_path');
                     });
-
+                    $('#expiration-date').val('');
```

Comment:
If from_path and to_url are being cleared as well, I feel like these should be in the same area as the others. My guess is in loadTableForSearch()?